round4/team-05.md

```diff
@@ -0,0 +1,162 @@
+# Chrome Extension Idea: [FlyHistory]
+
+## Authors
+
+Melanie Chen\
+Noah Erdogan\
+Jason Figueroa\
+Brady Zhou
+
+## Problem Statement
```

Comment:
Great proposal. You had a very clear outline of the problem and the value of the solution that you're offering. You also had a realistic risk analysis which is important.

My main question about this project is how exactly this extortion differs from the google search. From that I gather, it would store the flight search history, across multiple searches and multiple tables, is that correct?